madgraph_gridpack.py

```diff
@@ -17,6 +17,26 @@ def prepare_default_card(self):
         if glob.glob(f'{cards_path}/*_cuts.f'):
             os.system(f'cp {cards_path}/*_cuts.f {job_files_path}')
 
+
+    def get_proc_card(self):
+        """
+        Read proc card
+        Glue madspin card if it exists
+        """
+        if glob.glob(f'{cards_path}/*_proc_card.dat'):
+            with open(f'{cards_path}/*_proc_card.dat') as input_file:
+                proc_card = input_file.read()
+        else:
+            raise Exception(f'Could not find {cards_path} as process card')
+
+        if glob.glob(f'{cards_path}/*_madspin_card.dat'):
+            with open(f'{cards_path}/*_madpsin_card.dat') as input_file:
+                proc_card += '\n\n\n\n\n\n#### madspin_card.dat\n'
+                proc_card += input_file.read()
+
+        return proc_card
+ 
```

Comment:
madspin card depends on situation, add #### banner to split proc_card and madspin_card.dat parts